util/misc/fixInternalDocs

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
I don't think util/misc is included in the release tarball, won't that break `make documentation`?